regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -67,12 +66,13 @@ var CommentReviewView = Backbone.View.extend({
     this.$content.html(html);
     this.findElms();
 
+    this.$form.find('[name="comments"]').val(JSON.stringify(commentData));
+
     this.preambleHeadView = new PreambleHeadView();
     CommentEvents.trigger('comment:writeTabOpen');
   },
 
   serialize: function() {
```

Comment:
Given that this isn't used for general serialization anymore, should we rename it or inline it into `preview`?